foreman/foreman.spec

```diff
@@ -835,62 +900,6 @@ EOF
 rm -rf %{buildroot}
 
 %files
-%defattr(-,root,root,0755)
-%doc CHANGELOG Contributors LICENSE README.md VERSION
-%dir %{_datadir}/%{name}
-%{_datadir}/%{name}/app
-%exclude %{_datadir}/%{name}/app/assets
-%exclude %{_datadir}/%{name}/script/%{name}-debug.d
-%dir %{_datadir}/%{name}/bundler.d
-%exclude %{_datadir}/%{name}/bundler.d/development.rb
-%{_datadir}/%{name}/bundler.d/facter.rb
-%{_datadir}/%{name}/bundler.d/jsonp.rb
-%exclude %{_datadir}/%{name}/bundler.d/openid.rb
-%exclude %{_datadir}/%{name}/bundler.d/test.rb
-%{_datadir}/%{name}/bin
-%{_datadir}/%{name}/config*
-%{_datadir}/%{name}/db
-%{_datadir}/%{name}/extras
-%{_datadir}/%{name}/Gemfile.in
-%{_datadir}/%{name}/lib
-%{_datadir}/%{name}/locale
-%{_datadir}/%{name}/log
-%{_datadir}/%{name}/migrate
-%{_datadir}/%{name}/plugins
-%{_datadir}/%{name}/public
-%{_datadir}/%{name}/Rakefile
-%{_datadir}/%{name}/script
-%{_datadir}/%{name}/seeds.*
-%attr(700,%{name},%{name}) %{_datadir}/%{name}/.ssh
-%{_datadir}/%{name}/tmp
-%{_datadir}/%{name}/VERSION
-%{_initrddir}/%{name}
-%{_sbindir}/%{name}-rake
-%{_sbindir}/%{name}-tail
-%{_mandir}/man8
-%config(noreplace) %{_sysconfdir}/%{name}
-%ghost %attr(0640,root,%{name}) %config(noreplace) %{_sysconfdir}/%{name}/encryption_key.rb
-%config(noreplace) %{_sysconfdir}/sysconfig/%{name}
-%config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
-%config %{_sysconfdir}/cron.d/%{name}
-%{_sysconfdir}/rpm/macros.%{name}
-%attr(-,%{name},%{name}) %{_localstatedir}/lib/%{name}
-%attr(750,%{name},%{name}) %{_localstatedir}/log/%{name}
-%attr(750,%{name},%{name}) %{_localstatedir}/log/%{name}/plugins
-%attr(-,%{name},%{name}) %{_localstatedir}/run/%{name}
-%attr(-,%{name},root) %{_datadir}/%{name}/config.ru
-%attr(-,%{name},root) %{_datadir}/%{name}/config/environment.rb
-# Symlink to /etc, EL6 needs attrs for ghost files, Fedora doesn't
-%if 0%{?rhel} == 6
-%ghost %attr(0777,root,root) %{_datadir}/%{name}/config/initializers/encryption_key.rb
-%else
-%ghost %{_datadir}/%{name}/config/initializers/encryption_key.rb
-%endif
-%ghost %attr(0640,root,%{name}) %config(noreplace) %{_datadir}/%{name}/config/initializers/local_secret_token.rb
-# Only need tmpfiles on systemd (F17 and up)
-%if 0%{?rhel} > 6 || 0%{?fedora} > 16
-%{_prefix}/lib/tmpfiles.d/%{name}.conf
-%endif
 
 %pre
```

Comment:
they are meant to be in foreman, not core, moved the pre/post/preun depednencies